admin_tests/nodes/test_serializers.py

```diff
@@ -7,7 +7,6 @@
 
 
 class TestNodeSerializers(AdminTestCase):
-
     def test_serialize_node(self):
```

Comment:
Can you update this test (and the `simple_node` test in `users/test_serializers.py`) to include a check for `'deleted'`?